src/sdk/pynni/nni/compression/torch/compressor.py

```diff
@@ -468,45 +466,45 @@ class Quantizer(Compressor):
     """
 
     def __init__(self, model, config_list):
-        super().__init__(model, config_list)
+        super().__init__(model, config_list, None)
```

Comment:
better to give default value `None` for optimizer in `Compressor`

Reply:
Done.